request_test.go

```diff
@@ -55,6 +55,19 @@ func TestFieldsFromRequestNoError(t *testing.T) {
 			t.Errorf("Expected [foo bar] but got %v", fields)
 		}
 	}
+	{
```

Comment:
I do not see the value added by this test, it tests the same code path as the previous test

Reply:
FieldsFromRequest does not care about the values of select foo,foo is handled exactly as foo,bar. If you prefer to keep it, you can leave it here, It does not make a difference...

Reply:
Ah you're right. Sorry, trying to do multiple things :) I'll remove this then.